src/G2/Execution/SymToCase.hs

```diff
@@ -58,23 +55,36 @@ createCaseExpr :: TyVarEnv
                -> [DataCon]
                -> (Id, Expr, [PathCond], ExprEnv, NameGen)
 createCaseExpr tv bi maybeC binder ti kv tenv eenv ng dcs =
```

Comment:
Do we still use createCaseExpr anywahere? If not, can cut (and maybe rename createCaseExprInsertless to just createCaseExpr)?